packages/client/tests/functional/_utils/setupFilesAfterEnv.ts

```diff
@@ -3,5 +3,7 @@ process.env.DATABASE_URI_sqlite = 'file:dev.db'
 process.env.DATABASE_URI_mongodb = process.env.TEST_MONGO_URI
 process.env.DATABASE_URI_postgresql = process.env.TEST_POSTGRES_URI
 process.env.DATABASE_URI_mysql = process.env.TEST_MYSQL_URI
+process.env.DATABASE_URI_cockroachdb = process.env.TEST_COCKROACH_URI
```

Comment:
Nice work on adding these, I noticed however you arent testing either, can we add coverage for both of these providers in this test?